extensions/amp-analytics/0.1/test/test-visibility-impl.js

```diff
@@ -507,13 +505,24 @@ describe('amp-analytics.visibility', () => {
       unobserveSpy = sandbox.stub();
       callbackSpy1 = sandbox.stub();
       callbackSpy2 = sandbox.stub();
-      sandbox.stub(ampdoc.win, 'IntersectionObserver', callback => {
-        inObCallback = callback;
-        return {
-          observe: observeSpy,
-          unobserve: unobserveSpy,
-        };
-      });
+
+      if (inob.nativeIntersectionObserverSupported(ampdoc.win)) {
```

Comment:
During local test, I believe it is always the case that nativeIntersectionObserver is supported. Also I fail to find where the usage of IntersectionObserver logic was tested, don't quite understand why are we having this https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/test/test-visibility-impl.js#L116